File: src/server/Models/Meeting.cs
using System;
using System.Collections.Generic;

namespace WebRTCVideoCallAssistant.Server.Models
{
    public partial class Meeting
    {
        public int Id { get; set; }
        public int CreatedById { get; set; }
        public int CreatedForId { get; set; }
        public string UserConnId { get; set; } = null!;
        public string CustomerConnId { get; set; } = null!;
        public string UserSlug { get; set; } = null!;
        public string CustomerSlug { get; set; } = null!;
        public DateTime StartsAt { get; set; }
        public DateTime CreatedAt { get; set; }
        public int? StatId { get; set; }

        public virtual User CreatedBy { get; set; } = null!;
        public virtual Customer CreatedFor { get; set; } = null!;
        public virtual Stat? Stat { get; set; }
    }
}

File: src/client/src/app/models/meeting.ts
import { Customer } from "./customer";
import { Stat } from "./stat";
import { User } from "./user";

export interface Meeting {
    id: number;
    createdById: number;
    createdForId: number;
	userConnId: string;
	customerConnId: string;
    slug: string;
    startsAt: string;
    createdAt: string;
    statId:number;
    stat:Stat;
    createdBy:User;
    createdFor:Customer;
}

export interface MeetingCreate {
    createdById: number;
    createdForId: number;
    startsAt: string;
}
